amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1925,36 +1925,50 @@ vx_node createNode(vx_graph graph, vx_enum kernelEnum, vx_reference params[], vx
     return node;
 }
 
-// looks like following code is not used
-#if 0//ENABLE_OPENCL
-
-vx_status createGraphHandle(vx_node node, RPPCommonHandle **pHandle)
-{
-    RPPCommonHandle *handle = NULL;
+vx_status createRPPHandle(vx_node node, vxRppHandle **pHandle, Rpp32u batchSize, Rpp32u deviceType) {
```

Comment:
adjust the spacing here

Reply:
I am not seeing any spacing issues in this line